.github/workflows/ci-github.yml

```diff
@@ -17,14 +17,21 @@ jobs:
         include:
           - name: macOS
             os: macos-latest
+            ocaml: 4.14
             test_disabled:
             # We exclude the coq-core package
             packages: ./coqide-server.opam ./rocq*.opam
           - name: Windows
             os: windows-latest
+            ocaml: 4.14
             test_disabled: misc coq-makefile precomputed-time-tests
             # we don't support rocqide on Windows yet, so we filter it, see #20631
             packages: ./coqide-server.opam ./rocq-*.opam
+          - name: Linux (latest OCaml)
+            os: ubuntu-latest
+            ocaml: 5
```

Comment:
I'd recommend being more precise here, otherwise the CI can become non-deterministic on a new ocaml, which can be confusing for users.